app.rb

```diff
@@ -0,0 +1,111 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rentals'
```

Comment:
- [ ] Kindly use `require_relative 'rental` instead of `...rentals` to avoid this kind of error 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/218267128-0e1edb26-dcc4-4c5e-8eb9-95697cda370d.png)